dftimewolf/lib/containers/containers.py

```diff
@@ -353,6 +353,22 @@ def __init__(self, hostname: str, platform: str = 'unknown') -> None:
     self.platform = platform
 
 
+class GrrHostFlowPair(interface.AttributeContainer):
```

Comment:
Given that Flows are always associated with a Host (as they are unique *per host*), I think we can just rename this to `GrrHostFlow` or even `GrrFlow`.